dev_docs/tbs.md

```diff
@@ -61,3 +61,13 @@ As mentioned above, APM Servers will buffer events locally until a sampling deci
 For this purpose, we use [BadgerDB](https://github.com/dgraph-io/badger). BadgerDB is designed for
 fast writes, which is important for our use case: we write all events here, and read out only the
 sampled ones. All non-sampled events will be removed from the store through TTL expiry.
+
+BadgerDB doesn't provide a setting to limit disk usage, which is not ideal, since APM Server will
+most likely be running with a limit on storage and allowing BadgerDB to consume all disk space
+will eventually cause the APM Server to malfunction. For this reason, we have implemented a storage
+limit based on the size of the Badger database as reported by the SDK. This accounting has some lag
+of up to 1 minute and may allow the database to grow over the configured limit.
```

Comment:
I would say the first half of this paragraph can be dropped. The fact that badger doesn't have a limit isn't (I am guessing) important to the user; they can now set a limit which we roughly enforce. Maybe something like:

>We have implemented a configurable storage
limit based on the size of the Badger database as reported by the SDK. This accounting has some lag
of up to 1 minute and may allow the database to grow over the configured limit.

Reply:
Ah, and maybe mentioning that the limit can be disabled by setting its value to 0